packages/storybook/jest.config.js

```diff
@@ -5,5 +5,7 @@ module.exports = {
     '^.+\\.[tj]sx?$': 'ts-jest',
   },
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
-  globals: { 'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json' } },
+  globals: {
+    'ts-jest': { tsConfig: '<rootDir>/tsconfig.spec.json', diagnostics: false },
```

Comment:
so I don't know what happened bud I had to turn type checking within ts-jest off, because suddenly (after I created migration) majority of test was failing on type errors

Reply:
https://github.com/nrwl/nx/pull/3455#discussion_r470798145